assembly/pom.xml

```diff
@@ -21,12 +21,12 @@
   <dependencies>
     <dependency>
       <groupId>org.tachyonproject</groupId>
-      <artifactId>tachyon-core-common</artifactId>
+      <artifactId>tachyon-core-server</artifactId>
       <version>${project.version}</version>
     </dependency>
     <dependency>
       <groupId>org.tachyonproject</groupId>
-      <artifactId>tachyon-core-server</artifactId>
+      <artifactId>tachyon-keyvalue-server</artifactId>
```

Comment:
now assembly jar contains keyvalue servers, and we enable/disable keyvalue feature by using `tachyon.keyvalue.enabled` property